e2e/framework/helper.go

```diff
@@ -429,18 +475,45 @@ func NewTestServers(t *testing.T, num int, conf func(*TestServerConfig)) []*Test
 		srv := NewTestServer(t, dataDir, conf)
 		srv.Config.SetBootnodes(bootnodes)
 
-		if genesisErr := srv.GenerateGenesis(); genesisErr != nil {
-			t.Fatal(genesisErr)
-		}
+		srvs = append(srvs, srv)
+	}
 
-		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-		defer cancel()
+	errors := NewAtomicErrors(len(srvs))
 
-		if err := srv.Start(ctx); err != nil {
-			t.Fatal(err)
-		}
+	var wg sync.WaitGroup
 
-		srvs = append(srvs, srv)
+	for i, srv := range srvs {
+		wg.Add(1)
+
+		i, srv := i, srv
+
+		go func() {
+			defer wg.Done()
+
+			if err := srv.GenerateGenesis(); err != nil {
+				errors.Append(fmt.Errorf("server %d failed genesis command, error=%w", i, err))
+
+				return
+			}
+
+			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
You can use DefaultTimeout for 10*time.Second

Reply:
I've swapped all occurrences where we use a 10s timeout for the `DefaultTimeout`:

[99d626e](https://github.com/0xPolygon/polygon-edge/pull/526/commits/99d626e69b20db13a7d5a7b1e28fb9b428aa25a3)